client/my-sites/checkout/src/components/checkout-main-content.tsx

```diff
@@ -508,6 +511,18 @@ export default function CheckoutMainContent( {
 	}
 
 	const nextStepButtonText = translate( 'Continue to payment', { textOnly: true } );
+	const canEditStep = () => {
```

Comment:
I think I'll update this to be `canEditPaymentStep` since the point of these changes is to allow any function/boolean to be passed to the CheckoutStep component's `canEditStep` prop for _any_ step, by naming the helper function so generically it just adds confusion.

Reply:
Yeah, that makes a lot of sense.